databricks_cli/sdk/api_client.py

```diff
@@ -110,12 +121,18 @@ def perform_query(self, method, path, data = {}, headers = None):
 
         with warnings.catch_warnings():
             warnings.simplefilter("ignore", exceptions.InsecureRequestWarning)
-            if method == 'GET':
+            if method in self.restful_methods:
+                restful_method = method
+                if method == 'REST-GET':
+                    restful_method = 'GET'
```

Comment:
Why is this necessary? I see you don't typically pass in `data` anyways. If you don't won't `method = GET` do the same thing?

Reply:
I just tried to keep the restful API separate from the existing API where we try and create the query parameters for GET today. Didn't want to cause an unintended change due to how APIs are being designed now vs earlier.

Reply:
Guys - Please feel free to fix this after it has been merged. I've no time to look at it and test it now. At this point I don't care if it gets approved or not, because I was trying to do this coz Eng didn't have capacity and CLI is often overlooked for new APIs.